src/Group.php

```diff
@@ -31,6 +31,25 @@ public function __construct(?string $prefix = null, ?callable $callback = null,
         return $factory($prefix, $routes);
     }
 
+    final public function setContainer(ContainerInterface $container): Group
```

Comment:
I think it must be named as `withContainer` (named strategy for fluent setters).